hathor/cli/util.py

```diff
@@ -157,6 +157,9 @@ def setup_logging(
     else:
         handlers = ['pretty']
 
+    # Flag to enable debug level for both sync-v1 and sync-v2.
+    debug_sync = False and debug
```

Comment:
Why keep this if it always evaluates to `False`?

Reply:
I thought it might be useful when someone is debugging in the future. Just change this flag and both sync-v1 and sync-v2 will log on DEBUG level.

Reply:
Ah, ok!